clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystemCore.java

```diff
@@ -206,4 +206,43 @@ void setState(TachyonFile file, SetStateOptions options) throws IOException,
    */
   boolean unmount(TachyonURI tachyonPath, UnmountOptions options) throws IOException,
       TachyonException;
+
+  /**
```

Comment:
The convention in the API is to use setState for setting the path properties such as owner, group, and permissions.